app/controllers/hubstats/deploys_controller.rb

```diff
@@ -30,11 +30,19 @@ def show
     end
 
     def create
-      @deploy = Deploy.new(params[:deploy])
-      if @deploy.save
-        redirect_to :action => 'index'
-      else
+      if params[:deployed_by].nil? || params[:git_revision].nil? || params[:repo_name].nil?
         render :nothing => true, :status => 400
+      else
+        @deploy = Deploy.new
+        @deploy.deployed_at = params[:deployed_at]
+        @deploy.deployed_by = params[:deployed_by]
+        @deploy.git_revision = params[:git_revision]
+        @deploy.repo_id = Hubstats::Repo.where(full_name: params[:repo_name]).first.id.to_i
```

Comment:
Does this throw an exception if a known Repo is not found?  I'm curious what `.first.id.to_i` will do when `where` returns an empty collection/array.